app/models/force/application.rb

```diff
@@ -58,6 +64,11 @@ class Application < Force::ObjectBase
       { custom_api: 'hasCompletedHomebuyerEducation', domain: 'has_completed_homebuyer_education', salesforce: 'Has_Completed_Homebuyer_Education' },
       { custom_api: 'hasLoanPreapproval', domain: 'has_loan_preapproval', salesforce: 'Has_Loan_Pre_approval' },
       { custom_api: 'lendingAgent', domain: 'lending_agent', salesforce: 'Lending_Agent' },
+      { custom_api: 'shortFormPreferences', domain: 'preferences', salesforce: '?', object: Force::Preference},
```

Comment:
It would be nice to use this info about "object" to do nested mapping, but since some of them are a list of the object and some of them aren't, and every nested object has a bit of a special case I haven't yet